PWGCF/TwoParticleCorrelations/Tasks/CMakeLists.txt

```diff
@@ -27,8 +27,3 @@ o2physics_add_dpl_workflow(identifiedbf-filter-qa
                            SOURCES identifiedbfqa.cxx
                            PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore O2Physics::PWGCFCore
                            COMPONENT_NAME Analysis)
-
-o2physics_add_dpl_workflow(twopartcorr-efficiency-qc
-                           SOURCES efficiencyAndQc.cxx
-                           PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore O2Physics::PWGCFCore
-                           COMPONENT_NAME Analysis)
```

Comment:
This shows that the latest version of O2Physics has not been used for introducing the changes
The `efficiencyAndQc.cxx`  source file has been part of the O2Physics tree since February 25 (https://github.com/AliceO2Group/O2Physics/commits/master/PWGCF/TwoParticleCorrelations/Tasks/efficiencyAndQc.cxx) but in here is being removed due to changes done on a version older than that date

Reply:
![Screenshot from 2024-05-09 08-00-15](https://github.com/AliceO2Group/O2Physics/assets/25014127/7667d39b-5f59-49dc-beb0-686cf9bbf339)
Hello Victor. Here's what I get when I do "git status". However, I am now unable to push to the forked repository that I initiated the PR from from my local repository because I implemented the automatically generated PR from alibuild for formatting on that repository. I get the following error message when I do "git push origin master"

brian@brian-Latitude-7440:~/alice/O2Physics$ git push origin master 
To github.com:bghanley1995/O2Physics.git
 ! [rejected]          master -> master (non-fast-forward)
error: failed to push some refs to 'github.com:bghanley1995/O2Physics.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

When I use "git pull origin master" it puts me 11 commits behind AliceO2Group. 

I am trying to find the instructions on contributing via git to see if there's a solution to this issue or if I have somehow set up my git repository incorrectly and need to start over fresh.

Reply:
Those 11 commits from upstream are missing in your branch because when you tried to merge them into your branch, you modified them which created new 11 commits in your branch. You should reset your branch back to your last development commit and merge the upstream changes cleanly. Then you will have to force the push to your fork remote.